.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+## Communication
+
+We use [slack](https://goldcardcommunity.slack.com/) for talking about all `#website` matters.
```

Comment:
If we list that one, we should be aware that this will be public.
 
Will guest be allowed to that channel? Someone at g0v just asked me where things get coordinated. I think having "guests" to that channel might be great?

Reply:
By default they need an invite until we let them in if I'm not wrong. 

I like the guest model, it could be a good taste of what it's like to be in the gold card community.